docker-compose.yaml

```diff
@@ -0,0 +1,28 @@
+version: '3'
+services:
+  server:
+    image: node:latest
```

Comment:
You could add client services to the yaml file, so that docker compose up will run the live server.